src/pages/LandingPage/Home/Home.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import AuthNav from '../../../layout/LandingPage/nav/AuthNav';
+
+// import bicycleImage from '../../../components/asset/bicycle.jpg';
```

Comment:
-[ ] Kindly, delete the old commented code present in this file in order to preserve the repo's cleanliness. 👍